gemmbench/problems.py

```diff
@@ -1051,6 +1081,14 @@ def get_matching_configs(
             continue
         if not tag_re.match(tag):
             continue
+        # The raw_accumulators arg means "test configs where the result element
+        # type is different from what it would be in the default mode".
+        # We can't just test for (result_element_type == accumulator_element_type),
+        # as that would cause e.g. f32 matmuls to be omitted in the default mode.
```

Comment:
Should we add either a warning here that tells the user they are trying to do a raw_accumulators run where the `config.operand_element_type== get_default_accumulator_element_type(config.operand_element_type)` which we don't run.

Reply:
Such a warning would print every time the user passes `--raw_accumulators`, right?  I was thinking that since this flag is non-default, it's OK for it to have slightly suprising semantics of omitting the cases that happen to be already covered by the default mode.  I cared more about keeping the default mode unsurprising (including if in the future we add f32 benchmarks) and avoiding overlap between the two modes redundantly covering the same cases (would be wasteful if running both modes one after the other).

Reply:
It would only print if they are running --raw_accumulators with f32 or i32 input configs. And if that is the case, might be worth to have a small print or warning to let them know they are being skipped because of this, but fine with either way

Reply:
I went ahead and merged this because I want to use i8 in my experiments